### ISSUE TITLE
Send real-time event stream to amplitude via their HTTP API

### ISSUE THREAD

**Issue by rfk** [mozilla/fxa-auth-server#2044]
Separate card for sending the results from #2043 over to amplitude via their HTTP API.

**Comment by davismtl** [comment]
Should this be in train 95?

**Comment by philbooth** [comment]
> Should this be in train 95?

I don't think so. My priority for 95 is mapping from our existing events to the taxonomy and this issue still has a couple of (for me) unknowns that need to be figured out.

**Comment by philbooth** [comment]
Noting this down now, but I haven't probably dug into it yet because dinner, so I might be jumping the gun. But do we need to consider rate-limiting at the amplitude end?

I figured it would be fun to run the content server functional tests to fill the dev amplitude project with loads of events in quick time (all I'd done prior to that was a few manual sign-ins etc, which worked well). During the course of the functional tests, I appeared to trip the rate-limiter a few times (I say appear because I didn't have any proper logging, I'll debug it properly tomorrow).

The docs don't mention rate-limiting from what I can see so maybe it wasn't that. But if it was, perhaps we need to send events in batches rather than as they come in, which is what I'd naively assumed we'd be doing. Not sure if logging 2.0 can work like that though, invoking us at timed intervals with the log lines grouped into an array/tuple? @jbuck, any ideas?

Anyway, perhaps I'm getting ahead of myself, I'll know more tomorrow.
